integration-tests/slack/slack-notify-channels.json

```diff
@@ -0,0 +1,113 @@
+{
+  "pass": [
+    {
+      "name": "dsde-qa",
+      "id": "C53JYBV9A"
+    }
+  ],
+  "fail": [
+    {
+      "name": "dsde-qa",
+      "id": "C53JYBV9A"
+    },
+    {
+      "name": "dsp-interactive-analysis",
+      "id": "CA3NP1733",
+      "tests": [
+        {
+          "name": "run-analysis"
+        },
+        {
+          "name": "run-rstudio"
+        },
+        {
+          "name": "analysis-context-bar"
+        },
+        {
+          "name": "run-notebook"
+        }
+      ]
+    },
+    {
+      "name": "dsp-batch",
+      "id": "C1EH66VCM",
+      "tests": [
+        {
+          "name": "find-workflow"
+        },
+        {
+          "name": "import-dockstore-workflow"
+        },
+        {
+          "name": "run-workflow"
+        },
+        {
+          "name": "run-workflow-on-snapshot"
+        },
+        {
+          "name": "run-catalog-workflow"
+        }
+      ]
+    },
+    {
+      "name": "dsp-workspaces",
+      "id": "C02LMB1PQ6R",
```

Comment:
Please change this one to C03F21QEWV7

![image](https://user-images.githubusercontent.com/484484/181253903-e4e318b3-f58c-4b29-a1c1-1610d824a8b2.png)

Reply:
Done. Thanks.